server-spi/src/main/java/org/keycloak/provider/ConfiguredProvider.java

```diff
@@ -24,6 +24,26 @@
  * @version $Revision: 1 $
  */
 public interface ConfiguredProvider {
+
+    /**
+     * Returns the category for this provider so that it can be used to group it together
+     * with others when rendering forms.
+     *
+     * @return the provider category. Can be {@code null}.
+     */
+    default String getDisplayCategory() {
```

Comment:
Making these methods `default` method helps to not break existing implementations while still providing a reasonable default.